app/commands/v2/import.rb

```diff
@@ -0,0 +1,98 @@
+module Commands
+  module V2
+    class Import < BaseCommand
+      def call
+        unless UuidValidator.valid?(payload[:content_id])
+          raise CommandError.new(
+          code: 422,
+          error_details: {
+            error: {
+              code: 422,
+              message: "Content id not valid",
+              fields: "content_id",
+            }
+          })
+        end
+
+        delete_all(payload[:content_id])
```

Comment:
This could cause a content store inconsistency. If the items with this content_id had a base_path that didn't exist in the import the content_stores would have an orphaned item(s)